components/FetchedDogsContainer/FetchedDogsContainer.module.css

```
@@ -52,17 +31,32 @@
 .dogImgTitle {
   position: relative;
   bottom: 2rem;
-  left: 3rem;
-  background: rgba(225, 0, 255, 0.692);
+  left: 0rem;
```

If you're setting no value, unit is not required. Moreover it's good practice to not provide unit in such case
```suggestion
  left: 0;
```